extensions/blocks/pinterest/pinterest.php

```diff
@@ -180,10 +218,28 @@ function render_amp_pin( $attr ) {
  * @return string
  */
 function load_assets( $attr, $content ) {
+	if ( ! jetpack_is_frontend() ) {
+		return $content;
+	}
 	if ( Blocks::is_amp_request() ) {
 		return render_amp_pin( $attr );
 	} else {
+		$url  = $attr['url'];
+		$type = pin_type( $url );
+
+		if ( ! $type ) {
+			return null;
```

Comment:
We should return an empty string here, since `render_callback` expects a string that will replace the original block content. It probably can handle `null` values correctly, but better safe than sorry.

```suggestion
			return '';
```